tests/integration-tests/conftest.py

```diff
@@ -1192,3 +1192,34 @@ def _pcluster_ami_without_standard_naming(version):
     if ami_id:
         client = boto3.client("ec2", region_name=region)
         client.deregister_image(ImageId=ami_id)
+
+
+@pytest.fixture()
+def ami_copy(region):
+    """
+    Define a fixture to manage the copy and deletion of AMI.
+
+    This AMI is used to test head node and compute node AMI update
+    """
+    copy_ami_id = None
+    client = boto3.client("ec2", region_name=region)
+
+    def _copy_image(image_id, test_name):
+        nonlocal copy_ami_id
+        copy_ami_id = client.copy_image(
+            Name=f"copyed_image_{test_name}", SourceImageId=image_id, SourceRegion=region
```

Comment:
in case the image is not cleaned-up for a previous run or in case tests run in parallel with the same name (for any reason) this would fail.

one option is to add a random string in the name. is there a way to leverage the AMI cleanup logic in Jenkins in case the clean-up is not executed bu tests?

Reply:
random string is added in test_name. For AMI cleanup pipeline, we only clean up ami with name matching to *aws-parallelcluster* regex.

Reply:
> we only clean up ami with name matching to aws-parallelcluster regex

is there a reason we can't use that name format here?

Reply:
add tag for copied ami to leverage cleanup_ami pipeline